staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/start.go

```diff
@@ -78,6 +79,9 @@ func NewCommandStartCustomResourceDefinitionsServer(out, errOut io.Writer, stopC
 }
 
 func (o CustomResourceDefinitionsServerOptions) Validate(args []string) error {
+	if errs := o.RecommendedOptions.Validate(); len(errs) != 0 {
+		return errors.NewAggregate(errs)
```

Comment:
can we also collect all errors in an error slice and then aggregate in the last line?